lib/src/model/tournament/tournament.dart

```diff
@@ -228,8 +228,15 @@ Tournament _tournamentFromPick(RequiredPick pick) {
 }
 
 Tournament _updateTournamentFromPartialPick(Tournament tournament, RequiredPick pick) {
+  final newFeaturedGameId = pick('featured', 'id').asGameIdOrNull();
   return tournament.copyWith(
-    featuredGame: pick('featured').asFeaturedGameOrNull(),
+    // Sometimes a new FEN comes in via the websocket, but the API reload response
+    // still has the FEN of the previous move, leading to sync issues.
+    // So only copy the whole game here if it's a new ID.
```

Comment:
Without this logic, I saw some glitches where moves were taken back due to API and websocket being out of sync

Reply:
Have you checked how it works on the website? Given the reload mechanism I find it strange indeed to use both this and the websocket.

Reply:
I can check again and/or ask @ornicar, but I think the reload events are not tied to the events of the featured game, but rather to the score/standings changing

Reply:
No it's fine like that. You need the reload events to know there is a new game. I assume the server sends the `reload` socket message when a new featured game starts.

Reply:
Exactly, when there's a new featured game, we get it via `reload` and then send `startWatching` via the socket to get the updates for each move